### ISSUE TITLE
Resource for images

### ISSUE THREAD

**Issue by sunithapriya** [Wwwzff/Ship-Detection#12]
In order to test your model, one must need access to images. I could not find the test and train images to test your model. I would suggest adding the train and test images to your folder or providing a link from where the images can be downloaded.

**Comment by Wwwzff** [comment]
Thx for advice! Just inserted source link of the dataset on kaggle

**Comment by ethanhou99** [comment]
Thanks for the suggestion. All the dataset can be downloaded from https://www.kaggle.com/c/airbus-ship-detection/data. However, the training set is 26GB and test set is 2.12GB, it's inconvenient for you to do that. You can simply check the main.ipynb or watch our demo in the class.
